lib/webdriveragent.js

```diff
@@ -64,6 +65,14 @@ class WebDriverAgent {
   }
 
   async launch (sessionId) {
+    var useExistingWebDriverAgent = !!this.webDriverAgentUrl;
+    if (useExistingWebDriverAgent) {
+      log.info('Using provided WebdriverAgent at ${this.webDriverAgentUrl}');
```

Comment:
This needs to be in backticks, instead of single quotes, for the template literal to work. Can you also enclose `${this.webDriverAgentUrl}` within singles quotes?

```js
log.info(`Using provided WebdriverAgent at '${this.webDriverAgentUrl}'`);
```